recipes/cad1/task1/baseline/config.yaml

```diff
@@ -0,0 +1,33 @@
+path:
+  root: ???
```

Comment:
Will this throw an error? Should it perhaps be `./` so it runs in the current directory assuming people copy the file to where they have downloaded the data?

One thought is adding the ability to an entry point that is defined (see above comments) to generate a configuration file automatically from this default.

Reply:
Yes, it will throw an error. The readme mention they must set this parameter